sky/serve/load_balancing_policies.py

```diff
@@ -10,6 +10,10 @@
 
 logger = sky_logging.init_logger(__name__)
 
+# Define a registry for load balancing policies
+LB_POLICIES = {}
+DEFAULT_LB_POLICY = 'round_robin'
```

Comment:
```suggestion
DEFAULT_LB_POLICY = None
```
should this be none?

Reply:
If we define this as `None` this code section is not useful:
```
        if policy_name is None:
            policy_name = DEFAULT_LB_POLICY
            return LB_POLICIES[policy_name]()
```